docs/release-notes/2.33.0.md

```diff
@@ -0,0 +1,162 @@
+To ensure a smooth rollout, PMM HA will be introduced in phases, in line with your requirements and feedback, with PMM 2.33.0 being the starting point. 
+
+!!! caution alert alert-warning "Important"
+    PMM HA will be rolled out in three stages. Stage one will be included with PMM 2.33.0 and stages two and three will be rolled out next year.
+
+The following features/use cases will be covered as part of the various stages of HA in PMM:
+
+**Stage 1**
+
+As a first step toward preventing data loss, we have developed the following solution:
+
+**Prevent data loss with VM integration for short outages**
+
+If PMM Server goes down, metrics will be buffered on the monitored instance until PMM Server comes back online, which is when metrics will get pushed to PMM Server. Note that this only applies to PMM Clients which run in the so-called `push` mode. The same holds true when a brief network outage occurs on the client side.
+
+**Stage 2**
+
+As part of PMM HA stage two in HA we plan to implement the following:
+
+**HA data sources**
+
+As part of stage two we will let the users use external data sources therby decreasing dependency on the file system.
+
+
+**Stage 3**
+
+As part of PMM HA stage three we plan to implement the following:
+
+**Clusterized HA**
+
+Clustered PMM will be the focus of stage three. Detailed information will be included in the upcoming release notes.
+
+
+
+
+
+### Guided tour of Percona Alerting
+
+
+Following the recent improvements to PMM Alerting, we have added a short tutorial to help you explore our streamlined alerting system based on Grafana.
+The in-app tutorial automatically pops up when you first open the **Alerting** page to help you uncover the fundamentals of Percona Alerting.
+
+
+ 
+### Restore MongoDB backups easier
+
+Building on the significant improvements for MongoDB Backup Management introduced in the previous release. We are now simplifying the process for restoring physical MongoDB backups.
+Starting with this release, you can restore physical backups straight from the UI instead of using Percona Server for MongoDB. In addition, you no longer need to perform additional manual operations after restoring the physical Backup.
+
+### Components Upgrade
+
+- VictoriaMetrics has been upgraded from 1.77.1 to 1.82.1.
+- As part of PMM 2.33.0, grafana has been updated to 9.2.5, which fixes some critical vulnerabilities. For more information, see [grafana 9.2.5](https://grafana.com/blog/2022/11/08/security-release-new-versions-of-grafana-with-critical-and-moderate-fixes-for-cve-2022-39328-cve-2022-39307-and-cve-2022-39306/)
+
+
+
+## New Features
+
+[PMM-10889](https://jira.percona.com/browse/PMM-10889) - Starting with PMM2.33.0, while using the pmm-admin CLI command, you do not have to know the entire command. Just type the initial command and press Tab. The rest of the command will be autocompleted.
+
+Depending on the shell you are using, add the following to your shells .rc file for autocomplete to work:
```

Comment:
also indent issue, please move right so it would be under the feature, not on the same level